app/components/ProgressBar/index.tsx

```diff
@@ -1,17 +1,20 @@
 import React, { useMemo } from 'react';
 import { _cs } from '@togglecorp/fujs';
+import { IoInformationCircleOutline } from 'react-icons/io5';
 
 import styles from './styles.css';
 
 export interface Props {
     className?: string | undefined;
     barHeight: number;
```

Comment:
Either change this to different types or remove.

Reply:
Requested changes done